_source/android/01_getting_started.md

```diff
@@ -55,6 +55,9 @@ Set up the app's layout by opening `activity_main.xml` and replace the entire fi
 Finally, open `MainActivity.kt` and replace the class with this code:
 
 ```kotlin
+// use the namespace of your application as listed in the app-level `build.gradle.kts` under `android / namespace`:
+package com.example.myapplication
+
```

Comment:
A different approach than adding this line with the comment would be to change the instructions on 55.

So instead of `Finally, open MainActivity.kt and replace the class with this code:`, it could say something like

`Finally, open MainActivity.kt and replace everything in the class **except the package reference** with this code:`